### PR TITLE
Add Command list orders

### PR DESCRIPTION
**Pull request by maze508**
list orders of a particular person (through indexing)
- Temporary UI

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ListOrderCommand.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+     * @return A string representation of the order list.
+     */
+    private String createOrderListString(List<Order> orders) {
+        StringBuilder sb = new StringBuilder();
+        sb.append("Orders for the selected person:\n");
+
+        // Sort orders by date using a stream
+        List<Order> sortedOrders = orders.stream()
+                .sorted(Comparator.comparing(Order::getDate))
+                .collect(Collectors.toList());
```
Comment (guanquann) [inline src/main/java/seedu/address/logic/commands/ListOrderCommand.java:70]: Currently it looks like you are sorting by date in ascending order. I think sorting it by descending order would be better? So the latest orders are at the top.

File: src/main/java/seedu/address/logic/commands/ListOrderCommand.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+        for (int i = 0; i < sortedOrders.size(); i++) {
+            sb.append(i + 1).append(". ").append(sortedOrders.get(i).toString()).append("\n");
```
Comment (guanquann) [inline src/main/java/seedu/address/logic/commands/ListOrderCommand.java:73]: Would it be better if you split this line into multiple lines to reduce cluttering

File: src/test/java/seedu/address/model/order/DateTest.java [hunk 3]
```diff
@@ -53,4 +53,15 @@ public void hashcode() {
...
+
+    @Test
+    public void compareTo() {
+        Date earlierDate = new Date("2020-01-01");
+        Date laterDate = new Date("2020-02-01");
+
+        assertTrue(earlierDate.compareTo(laterDate) < 0, "Expected earlierDate to be less than laterDate");
+        assertTrue(laterDate.compareTo(earlierDate) > 0, "Expected laterDate to be greater than earlierDate");
+        assertEquals(0, earlierDate.compareTo(new Date("2020-01-01")),
+                "Expected dates with the same value to be equal");
```
Comment (guanquann) [inline src/test/java/seedu/address/model/order/DateTest.java:65]: Currently for all the test cases we have, there is no message for the test. Would it be better to remove them for  standarisation?

**Comment by sun-ruiheng** [comment]
LGTM

**Review by sun-ruiheng — APPROVED** [review]

**Pull request merged**